qr-code/src/test/java/com/adyen/checkout/qrcode/DefaultQRCodeDelegateTest.kt

```diff
@@ -265,6 +293,17 @@ internal class DefaultQRCodeDelegateTest(
         }
     }
 
+    @Test
+    fun `download qr image successfully`(): Unit = runBlocking {
+        val expectedResult = Result.success(Unit)
+        whenever(fileDownloader.download(anyString(), anyString(), anyString(), anyString())) doReturn expectedResult
+
+        val actualResult = delegate.downloadQRImage()
+
+        assertEquals(expectedResult.isSuccess, actualResult.isSuccess)
+        verify(fileDownloader).download(anyString(), anyString(), anyString(), anyString()).isSuccess
```

Comment:
You are verifying that `isSuccess` is called on the result of the `download`, why is that?

Reply:
same here